gradle/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.3.1-all.zip
```

Comment:
don't forget to check in the update gradle.jar file.

Reply:
you mean gradle-wrapper.jar? that one hasn't changed

Reply:
yeah `gradle-wrapper.jar` it definitely changes with each version.

Reply:
I edited the distributionUrl directly which is why there's no change. maybe the gradle wrapper command updates that jar

Reply:
updated.